index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Hammad's Portfolio </title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <div class="app-wrap">
+    <header>
+        <nav class="toolbar">
+            <div class="logo">
+                My Logo
+            
+            <a href="#" class="button">
+              <i class="fa fa-bars"></i>
+            </a> 
+        </div>
+        </nav>
+    </header>
+
+    <main class="page">
+        <section class="intro">
+            <h2 class="name">I'm Hammad</h2>
+            <h2 class="name"> Glad to see you!</h2>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
```

Comment:
- [ ] Please use flexbox for each part `header` and `section` to align its items.
- [ ] Kindly make sure you use the right margins, font sizes, and font weights, try to stick as much as possible to the given design.
- [ ] Please add your background image to this `section` and make it appear like the desgin.
Try his [link](https://www.w3schools.com/cssref/pr_background-position.asp) will help.

![image](https://user-images.githubusercontent.com/68796232/158703159-6de8aad0-bdfc-481e-b045-d4ac4efa575e.png) 
![image](https://user-images.githubusercontent.com/68796232/158703296-4348ec14-1801-4692-99c0-629a2fa9e477.png)